tests/crosstest/dubbo2kitex/conf/dubbogo.yaml

```diff
@@ -0,0 +1,11 @@
+dubbo:
+  registries:
+    demoZK:
+      protocol: zookeeper
```

Comment:
Is it ok to skip zookeeper here? In the tests/dubbo/... I added such config entry because I'm not familiar with dubbo's config.

Reply:
Yes, we can use url to connect to remote server directly.